<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>489</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>FPGA Object Classification Accelerator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>FPGA Object Classification Accelerator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_console">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Console</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>71</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_email_2">
         <property name="text">
          <string>DESL IP Address</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_computation">
         <property name="text">
          <string>1.1.X.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_email">
         <property name="text">
          <string>Video Board Connection</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_computation_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>DDR Board Connection</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_vga_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_notificationTimer">
         <property name="text">
          <string>Image Path</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_image">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_browse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>Send Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>image</zorder>
   <zorder>TitleLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_vga_start</zorder>
   <zorder>pushButton_computation_start</zorder>
   <zorder>pushButton_browse</zorder>
   <zorder>plainTextEdit_console</zorder>
   <zorder>pushButton_clear</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_send</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_computation_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedComputation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_vga_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedVGA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedComputation()</slot>
  <slot>clickedVGA()</slot>
  <slot>clickedBrowse()</slot>
  <slot>clickedSend()</slot>
  <slot>clickedClear()</slot>
 </slots>
</ui>
